qcfractal/interface/collections/dataset.py

```diff
@@ -917,22 +1045,31 @@ def get_keywords(self, alias: str, program: str, return_id: bool = False) -> Uni
             return self.client.query_keywords([kwid])[0]
 
     def add_contributed_values(self, contrib: ContributedValues, overwrite=False) -> None:
-        """Adds a ContributedValues to the database.
+        """
+        Adds a ContributedValues to the database. Be sure to call save() to commit changes to the server.
 
         Parameters
         ----------
         contrib : ContributedValues
             The ContributedValues to add.
         overwrite : bool, optional
             Overwrites pre-existing values
+
+        Raises
+        ------
+        ValueError:
+            If contrib's indexes do not match those of the dataset's entries
+        KeyError:
+            If values with the same name already exist, and overwrite is False.
```

Comment:
I usually do not write out `Raises`. Any time I do it feel awkward, seems to be a general policy. Since we have been neglecting them everywhere, perhaps a removal here is in order.